utils/github.go

```diff
@@ -6,14 +6,46 @@ package utils
 
 import (
 	"context"
+	"fmt"
 	"net/url"
 	"strconv"
 	"strings"
 
 	"github.com/google/go-github/v42/github"
+	"github.com/shurcooL/githubv4"
 	"github.com/tomnomnom/linkheader"
 )
 
+type GQLActor struct {
+	Login githubv4.String
+}
+
+type GQLReviewThread struct {
+	Comments struct {
+		TotalCount githubv4.Int
+		Nodes      []struct {
+			Author GQLActor
+		}
+	} `graphql:"comments(first: 1)"`
+	IsOutdated githubv4.Boolean
+	IsResolved githubv4.Boolean
+	Path       githubv4.String
+}
```

Comment:
We shouldn't retrieve more information than required:

```suggestion
type GQLReviewThread struct {
	IsResolved githubv4.Boolean
}
```